shared-bindings/displayio/FourWire.c

```diff
@@ -52,7 +52,7 @@
 //|         :py:func:`displayio.release_displays` first, otherwise it will error after the first code.py run.
 //|
 //|         :param busio.SPI spi_bus: The SPI bus that make up the clock and data lines
-//|         :param microcontroller.Pin command: Data or command pin
+//|         :param microcontroller.Pin command: Data or command pin. When None, 9-bit words will be used.
```

Comment:
I think it'd be good to document what 9-bit words means above in a new paragraph. Throwing away 7 bits may surprise people.

Reply:
We only throw away bits at the end of the write call, otherwise they are packed into 8-bit chunks. I will  add better docs.